dojo/tools/solar_appscreener/parser.py

```diff
@@ -47,7 +47,7 @@ def get_findings(self, filename, test):
             if finding.line:
                 if not finding.line.isdigit():
                     finding.line = finding.line.split("-")[0]
-              finding.line = int(finding.line)
+                finding.line = int(finding.line)
```

Comment:
I think the error is still there, because after split if '-' will be on input string  got on output.  @damiencarol